manifests/r/RafaelCossovandeFrança/MenuTools/1.0.2.20250729/RafaelCossovandeFrança.MenuTools.installer.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.10.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.10.0.schema.json
+
+PackageIdentifier: RafaelCossovandeFrança.MenuTools

```

```suggestion
PackageIdentifier: navossoc.MenuTools
```
Because this is a GitHub release, it's preferred that the PackageIdentifier match the repository the release comes from. After changing this, the manifests will need to be moved to the matching directory and renamed.